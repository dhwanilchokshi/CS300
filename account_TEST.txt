913778223:valid:
913678225:valid:
913678233:valid:
914567234:suspended:30.45
912921123:NA:
