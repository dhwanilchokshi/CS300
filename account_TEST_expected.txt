2
2
913678233
0
3
